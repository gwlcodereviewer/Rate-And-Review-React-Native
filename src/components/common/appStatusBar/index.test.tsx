import React from 'react';
import renderer from 'react-test-renderer';
import AppStatusBar from '.';

let wrapper: renderer.ReactTestRenderer;

describe('render correctly', () => {
  wrapper = renderer.create(<AppStatusBar />);

  it('render correctly appStatusBar view', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
