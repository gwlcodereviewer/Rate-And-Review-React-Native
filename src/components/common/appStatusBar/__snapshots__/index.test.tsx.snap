// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render correctly appStatusBar view 1`] = `
<View
  bgColor="#F8F8F9"
  style={
    Array [
      Object {
        "backgroundColor": "#F8F8F9",
        "height": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      },
    ]
  }
/>
`;
